src/components/ha-label-badge.html

```diff
@@ -8,7 +8,7 @@
       display: inline-block;
       text-align: center;
       vertical-align: top;
-      margin-bottom: 16px;
```

Comment:
I think that the badge should never had a margin built-in to begin with. A margin on the outside should be part of whatever holds the label badge